kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/sink/EventBuilder.kt

```diff
@@ -0,0 +1,45 @@
+package streams.kafka.connect.sink
+
+import org.apache.kafka.connect.sink.SinkRecord
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class EventBuilder {
+    private val log: Logger = LoggerFactory.getLogger(EventBuilder::class.java)
+
+    private var batchSize: Int? = null
+    private lateinit var topics: Set<String>
+    private lateinit var sinkRecords: Collection<SinkRecord>
+
+    fun withBatchSize(batchSize: Int): EventBuilder {
```

Comment:
do we need the builder pattern here?
Why not just a class with batchsize and topics 
and then apply/prepare or similar method that takes the records